config/protocol.testnet.yml

```diff
@@ -1,5 +1,5 @@
 ProtocolConfiguration:
-  Magic: 877933390
+  Magic: 894710606
```

Comment:
`TestGetConfigFromContext` should be adjusted wrt this change.

Reply:
Yep, fixed.